armi/bookkeeping/db/tests/test_layout.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-r""" Tests for the db Layout and associated tools"""
+r""" Tests for the db Layout and associated tools."""
```

Comment:
Well, I didn't go through and delete ALL the extraneous ones, just the ones I happened to see while doing other things. But...

The `r` before `""` in Python is use if you use back tick quotes: r"This is a ``Reactor object"

Reply:
ohhhh so you were manually doing that. makes a lot more sense now!

does using an r-string allow you to use single-backtick rather than double-backtick to get the code formatting?